code-reviews/recipes/CSharp.md

```diff
@@ -1,22 +1,20 @@
 # C# Code Reviews
```

Comment:
># [](start = 3, length = 1)

should we have a section for testing tool guides?

Reply:
If we have that we should have a section for observability and security as well. 
I think the idea with the current play book structure is to have testing covered in the test section.  If we have it in both sections we will have duplicate content and if it is not kept up to date we may end up with conflicting documentation

Reply:
sounds good!